components/input/input.css

```diff
@@ -0,0 +1,41 @@
+.input {
+  position: relative;
+  margin: 2.5rem;
+  width: 50%;
+}
+
+.input > input {
+  width: 25rem;
+  height: 4rem;
+  padding: 0.2em;
+  border-radius: 0.5rem;
+}
+
+.input > div {
+  position: absolute;
+  top: -0.6em;
```

Comment:
Try not to use this selector. Add a separate class instead.

Reply:
Used separate classes now. Thank you.